packages/bot-web-ui/src/components/dashboard/dashboard-component/dashboard-component.tsx

```diff
@@ -17,6 +17,47 @@ type TDashboard = {
     setActiveTab: (param: number) => void;
 };
 
+type TDashboardMobileCommonProps = {
+    is_mobile: boolean;
+    has_dashboard_strategies: boolean;
+};
+
+const DashboardTitle = ({ is_mobile, has_dashboard_strategies }: TDashboardMobileCommonProps) => (
+    <div
+        className={classNames('tab__dashboard__header', {
+            'tab__dashboard__header--not-listed': !has_dashboard_strategies && is_mobile,
+        })}
+    >
+        <Text color='prominent' line_height='xxl' size={is_mobile ? 's' : 'm'} weight='bold'>
+            {localize('Load or build your bot')}
+        </Text>
+    </div>
+);
+
+const MobileIconGuide = ({ has_dashboard_strategies }: { has_dashboard_strategies: boolean }) => (
+    <MobileWrapper>
+        <div>
+            <Local />
+        </div>
+    </MobileWrapper>
+);
+
+const DashboardDescription = ({ is_mobile, has_dashboard_strategies }: TDashboardMobileCommonProps) => (
+    <div
+        className={classNames('tab__dashboard__description', {
+            'tab__dashboard__description__loaded--listed': has_dashboard_strategies && !is_mobile,
+            'tab__dashboard__description__loaded--not-listed':
+                !has_dashboard_strategies || (!has_dashboard_strategies && is_mobile),
+        })}
+    >
+        <Text color='prominent' line_height='s' size={is_mobile ? 'xxs' : 's'}>
+            {localize(
+                'Import bot from your computer or Google Drive, build it from scratch, or start with a quick strategy.'
+            )}
+        </Text>
+    </div>
+);
```

Comment:
Can we move them inside `< DashboardComponent >` to not to pass `is_mobile, has_dashboard_strategies` each time?

Reply:
@NikitK-deriv no, I took them out because I use them in different ways and in different places under certain conditions, the containers change according to the design

Reply:
+ to avoid duplicating of the code in one component